### PR TITLE
feat!: use zero questions mode by default.

### PR DESCRIPTION
**Pull request by KuznetsovRoman**
- use zero questions mode by default.
- "-y" argv replaced with "-v" with opposite meaning
- "noQuestions" replaced with "extraQuestions" with opposite meaning

### CODE REVIEW

File: docs/en/create-testplane.md [hunk 1]
```diff
@@ -14,17 +14,19 @@ Without specifying the path, project will be created in the current directory.
 
 If you already have a project at given path, the tool will try to guess used package manager.
 
-### No questions mode
+### Extra questions mode
 
-You can add `-y` or `--yes` argument to launch a tool in *no-questions* mode:
+By default, project will be created with zero questions.
```
Comment (sipayRT) [inline docs/en/create-testplane.md:19]: almost zero questions - there may be a question about downloading the npm-package :)

File: docs/en/create-testplane.md [hunk 2]
```diff
@@ -14,17 +14,19 @@ Without specifying the path, project will be created in the current directory.
...
-npm init testplane my-app -- -y
+npm init testplane my-app -- -v
 ```
 
 In this mode you won't be asked questions about desired plugins and packet manager.
 
-Default packet manager, used with `--yes` argument: `npm`
+Default packet manager, used without `--verbose` argument: `npm`
 
 Default plugins, installed with `--yes` argument:
```
Comment (sipayRT) [inline docs/en/create-testplane.md]: here we can remove part `installed with `--yes` argument`

File: docs/ru/create-testplane.md [hunk 3]
```diff
@@ -14,15 +14,21 @@ npm init testplane my-app
 
 Если по указанному пути уже имеется проект, инструмент попробует определить используемый пакетный менеджер, и будет использовать его для установки пакетов.
 
-### Режим «без вопросов»
+### Режим продвинутой установки
 
-Вы можете добавить аргумент `-y` или `--yes` чтобы запустить инструмент в режиме «без вопросов».
+По умолчанию, проект будет настроен без всяких вопросов.
+
+Вы можете добавить аргумент `-v` или `--verbose` чтобы запустить инструмент в режиме продвинутой установки, чтобы выбрать пакетный менеджер или сразу установить дополнительные плагины.
```
Comment (sipayRT) [inline docs/ru/create-testplane.md]: `чтобы, чтобы`
Для запуска режима продвинутой установки Вы можете добавить аргумент `-v` или `--verbose`. Это позволит выбрать ...

File: docs/ru/create-testplane.md [hunk 4]
```diff
@@ -82,15 +88,15 @@ createTestplane.run({
 
 **Обязательный параметр.**
 
-Консольный интерфейс по умолчанию обрабатывает данный путь и аргумент `--yes`.
-В этом коллбэке вам необходимо указать как минимум значения `path` и `noQuestions`:
+Консольный интерфейс по умолчанию обрабатывает данный путь и аргумент `--verbose`.
+В этом коллбэке вам необходимо указать как минимум значения `path` и `extraQuestions`:
```
Comment (sipayRT) [inline docs/ru/create-testplane.md]: указать, как минимум, значения

File: src/bin/create-testplane.ts [hunk 5]
```diff
@@ -17,11 +17,11 @@ const argv = yargs(hideBin(process.argv))
         description: "Language, which will be used to write Testplane tests",
         choices: ["ts", "js"],
     })
-    .option("yes", {
```
Comment (sipayRT) [inline src/bin/create-testplane.ts:20]: instead of removing this option we can set default value as `true` (we have many examples with `-y` option)

**Review by sipayRT — COMMENTED** [review]
you have a typo in your commit message - `feat!:`

**Review by sipayRT — APPROVED** [review]

File: src/bin/create-testplane.ts (hunk 5 above)
Comment (KuznetsovRoman) [reply src/bin/create-testplane.ts:20]: How would we turn it off then?
You still can use `-y` option (as well as `-f`, `-o`, `-o`, `-b`, `-a`, `-r`), it just does nothing

File: docs/en/create-testplane.md (hunk 2 above)
Comment (KuznetsovRoman) [reply docs/en/create-testplane.md]: I will replace it with "without `--verbose` argument"

**Pull request merged**